account-decoder/src/parse_token.rs

```diff
@@ -7,11 +7,18 @@ use spl_token_v1_0::{
 };
 use std::{mem::size_of, str::FromStr};
 
-// A helper function to convert spl_token_v1_0::id() as spl_sdk::pubkey::Pubkey to solana_sdk::pubkey::Pubkey
+// A helper function to convert spl_token_v1_0::id() as spl_sdk::pubkey::Pubkey to
+// solana_sdk::pubkey::Pubkey
 pub fn spl_token_id_v1_0() -> Pubkey {
     Pubkey::from_str(&spl_token_v1_0::id().to_string()).unwrap()
 }
 
+// A helper function to convert spl_token_v1_0::native_mint::id() as spl_sdk::pubkey::Pubkey to
+// solana_sdk::pubkey::Pubkey
+pub fn spl_token_v1_0_native_mint() -> Pubkey {
```

Comment:
As a follow-up sometime, I think we can remove these helper functions.  I setup things between spl and the monorepo to ensure that the solana-sdk version will always be the same.  The token-cli program I've been working on doesn't use these kind of helpers (and of course will 🍄 if there's ever a solana-sdk match)

Reply:
On master, I think the 1.3 vs 1.2 sdk versions still conflict. Unless I'm missing something?
```
if mint == &spl_token_v1_0::native_mint::id() {
            ^^ no implementation for `solana_sdk::pubkey::Pubkey == solana_sdk::pubkey::Pubkey`
```

Reply:
Might need a `cargo update`?  I've been building token-cli with this PR based off master.  It does tend to blow up on me if I don't do an `update` to ensure my Cargo.lock isn't forcing older versions of the sdk and other solana crates.  

We don't need to figure that out in this PR though.

Reply:
Hmm, I'm still seeing that error after a cargo update. It might be a difference in how directly the spl-sdk types are used here. Or maybe it's the equality check.
Anyway... issue for another day.